docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,95 @@
+# Running Flink-Job-Cluster in Kubernetes
+
+## Overview
+
+to run a Flink-Job-Cluster, you first have to deploy the Flink-Operator. For that, follow the [user_guide](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md)
+
+## Prerequisites
+
+This Chart requires you to have Flink Operator Running in your Cluster.
+It is advised to use at least the following image version of flink-operator: ```gcr.io/flink-operator/flink-operator:v1beta1-6```
+since it enables you to use annotations for the Flink-Job-Cluster components e.g. JobManager, TaskManager and Job.
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Prepare a Flink operator image. You should use at least ```gcr.io/flink-operator/flink-operator:v1beta1-6```.
+
+2. Deploy the Flink Operator Helm Chart, follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator)
+
+3. Prepare a Flink-Job-Cluster image. You should use ```flink:latest``` or ```flink:1.9.3``` which has been tested and confirmed to work with flink-job-cluster helm chart.
+
+3. Use the following command to install the flink-job-cluster chart:
+
+  ```bash
+  helm template --namespace [NAMESPACE] [RELEASE_NAME] [CHART] -f values.yaml |kubectl apply -f -
+  ```
+
+## Uninstalling the Chart
+
+To uninstall your release:
+
+  ```bash
+  helm template --namespace [NAMESPACE] [RELEASE_NAME] [CHART] -f values.yaml |kubectl delete -f -
+  ```
+
+## Monitoring
+
+The Flink-Job-Cluster comes with a PodMonitor ressource, which is the counter part to a ServiceMonitor.
+The PodMonitor will use pod labels and configure prometheus to scrape the flink-job-cluster metrics. Reason for using the podMonitor is simple, the flink-job-cluster does not deploy services.
+
+You can use the following [dashboard](https://grafana.com/grafana/dashboards/10369) in your grafana to monitor the flink-job-cluster.
+
+## Run Jobs with InitContainer
```

Comment:
This doesn't seem to be tightly relevant to this helm chart. You can briefly mention it and point to an example like https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/config/samples/flinkoperator_v1beta1_remotejobjar.yaml.